### ISSUE TITLE
Problem with gitignore on realease

### ISSUE THREAD

**Issue by ggoyanes** [usefulteam/jwt-auth#85]
Hi!

We are using jwt-auth plugin on our repositories and we are having a problem. 

Since vendor is included on the .gitignore file, when we include the plugin on our repositories (installing via wordpress panel), although the plugin cames with the vendor directory, we can not add the vendor directory to our repository, causing fatal errors when pulling on another machine.

Although we can fix it the problem with a composer install, I think the realeases shouldn't have the .gitignore file.

Would this be possible?

**Comment by rajanstha** [comment]
Yeah the releases should not contain the .gitignore file. Hope this will be fixed in next release.

**Comment by dominic-ks** [comment]
Hi @ggoyanes and @rajanstha,

Thanks for this, and yes it'll be sorted in the next release which is hopefully going to be this week sometime...
